src/constants/strings.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  venuedetailsGenderNeutralToilets: "Gender neutral toilets"
```

Comment:
I think the "d" in "details" should be uppercase. 🔍

Reply:
Good catch, fixed.